week5/will/taobao/index.css

```diff
@@ -260,30 +259,34 @@ footer {
   padding: 15px 0px;
 }
 
-.footer-firstline a {
+.footer-uperline a {
   padding: 0px 10px 0px 5px;
   border-right: 1px solid #485065;
-  margin-left: -2px;
+  margin-left: 0px;
 }
 
-.footer-secondline a {
+.footer-middleline a {
   margin: 0px 2px;
 }
 
-.footer-secondline span {
+.footer-middleline {
+  margin-left: 2px;
+}
+
+.footer-middleline span {
   color: #9c9c9c;
   font-size: 12px;
   margin-left: 30px;
 }
 
-.footer-thirdline p {
+.footer-lowerline p {
   color: #9c9c9c;
   font-family: "Microsoft YaHei", tahoma, arial, "Hiragino Sans GB",
     "\5b8b\4f53", sans-serif;
   font-size: 12px;
   display: inline;
   padding: 0px 10px 0px 6px;
-  margin-left: -4px;
+  margin-left: 0px;
```

Comment:
does it still look the same with a 0 margin to the left?

Reply:
fixed